fast/stages/2-networking-a-simple/main.tf

```diff
@@ -22,7 +22,11 @@ locals {
     for k, v in coalesce(var.service_accounts, {}) :
     k => "serviceAccount:${v}" if v != null
   }
-  spoke_connection = var.spoke_configs.peering_configs != null ? "peering" : "vpn"
+  spoke_connection = coalesce(
```

Comment:
I'm wondering whether we should delete this and directly use the variables in the code. It seems it doesn't really help reducing the complexity

Reply:
uh? I think this is a good way of doing it...

Reply:
I thought a bit about a potentially more elegant solution but I didn't find any. 
At some point you have an N-nary decision to make.

Reply:
OK, understood. lgtm. Feel free to resolve the comment.